frontend/src/APIClients/AuthAPIClient.ts

```diff
@@ -23,9 +23,7 @@ const login = async (
   }
 };
 
-const loginWithGoogle = async (
-  idToken: string,
-): Promise<AuthenticatedUser | string> => {
+const loginWithGoogle = async (idToken: string): Promise<AuthenticatedUser> => {
```

Comment:
this is probably the only change I'm a bit iffy on, was there a reason why you were returning a string as a type _ever_?